implementations/rust/ockam/ockam_command/src/project_admin/delete.rs

```diff
@@ -0,0 +1,135 @@
+use crate::shared_args::IdentityOpts;
+use crate::tui::{DeleteCommandTui, PluralTerm};
+use crate::{Command, CommandGlobalOpts};
+use async_trait::async_trait;
+use clap::Args;
+use colorful::Colorful;
+use console::Term;
+use miette::IntoDiagnostic;
+use ockam::Context;
+use ockam_api::cloud::email_address::EmailAddress;
+use ockam_api::cloud::project::{Project, ProjectsOrchestratorApi};
+use ockam_api::colors::color_primary;
+use ockam_api::fmt_ok;
+use ockam_api::nodes::InMemoryNode;
+use ockam_api::terminal::{Terminal, TerminalStream};
+
+/// Delete an Admin from a Project
+#[derive(Clone, Debug, Args)]
+#[command()]
+pub struct DeleteCommand {
+    /// Email of the Admin to add
+    #[arg(value_parser = EmailAddress::parse)]
+    email: Option<EmailAddress>,
+
+    /// Name of the Project
+    name: Option<String>,
+
+    /// Confirm the deletion without prompting
+    #[arg(long, short)]
+    yes: bool,
+
+    #[arg(long, short)]
```

Comment:
Let's remove the short option for every `--all` option, to be consistent with `reset`.